### PR TITLE
Closes #2767 Re-export config ahead of 2.8.0 release.

### LINKED ISSUE #2767: Re-export config ahead of 2.8.0 minor release.
We need to re-export config now that we've merged #2587 before we create any 2.8.x releases.

### PR DESCRIPTION
**Pull request by joeparsons**
## Description
Re-exported config in preparation for 2.8.0 now that we've merged #2587.

## Related issues

Closes #2767

## How to test

https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-2768.probo.build/

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [x] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by joeparsons** [comment]
Change record associated with the oembed eager loading changes: https://www.drupal.org/node/3275103

File: modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_background.yml [hunk 1]
```diff
@@ -20,6 +20,8 @@ content:
     settings:
       max_width: 0
       max_height: 0
+      loading:
+        attribute: eager
```
Comment (joeparsons) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_background.yml:24]:
```suggestion
        attribute: lazy
```

File: modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_large.yml [hunk 2]
```diff
@@ -41,6 +41,8 @@ content:
     settings:
       max_width: 0
       max_height: 0
+      loading:
+        attribute: eager
```
Comment (joeparsons) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_large.yml:45]:
```suggestion
        attribute: lazy
```

File: modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_medium.yml [hunk 3]
```diff
@@ -41,6 +41,8 @@ content:
     settings:
       max_width: 0
       max_height: 0
+      loading:
+        attribute: eager
```
Comment (joeparsons) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_medium.yml:45]:
```suggestion
        attribute: lazy
```

File: modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_natural_size.yml [hunk 4]
```diff
@@ -41,6 +41,8 @@ content:
     settings:
       max_width: 0
       max_height: 0
+      loading:
+        attribute: eager
```
Comment (joeparsons) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_natural_size.yml:45]:
```suggestion
        attribute: lazy
```

File: modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_small.yml [hunk 5]
```diff
@@ -41,6 +41,8 @@ content:
     settings:
       max_width: 0
       max_height: 0
+      loading:
+        attribute: eager
```
Comment (joeparsons) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_small.yml:45]:
```suggestion
        attribute: lazy
```

File: modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_square.yml [hunk 6]
```diff
@@ -41,6 +41,8 @@ content:
     settings:
       max_width: 0
       max_height: 0
+      loading:
+        attribute: eager
```
Comment (joeparsons) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.az_square.yml:45]:
```suggestion
        attribute: lazy
```

File: modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.default.yml [hunk 7]
```diff
@@ -40,6 +40,8 @@ content:
     settings:
       max_width: 0
       max_height: 0
+      loading:
+        attribute: eager
```
Comment (joeparsons) [inline modules/custom/az_media/config/install/core.entity_view_display.media.az_remote_video.default.yml:44]:
```suggestion
        attribute: lazy
```

**Review by bberndt-uaz — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Pull request merged**